spec/services/next_available_reservation_finder_spec.rb

```diff
@@ -25,4 +28,45 @@
       end
     end
   end
+
+  describe "daily booking instrument" do
+    let(:instrument) do
+      create(
+        :setup_instrument,
+        :always_available,
+        :daily_booking
+      )
+    end
+
+    describe "with an empty schedule" do
+      it "has a duration of 1 day when no minimum" do
+        expect(reservation.duration_mins.minutes).to eq 1.day
+      end
+
+      describe "reservation duration" do
+        before { instrument.update(min_reserve_days: 4) }
+
+        it "has a correct duration in minutes" do
+          expect(reservation.duration_mins.minutes).to eq instrument.min_reserve_days.days
+        end
+
+        it "has a correct duration in days" do
+          expect(reservation.duration_days).to eq instrument.min_reserve_days
+        end
+      end
+
+      it "returns a schedule as soon as possible" do
+        next_available_time = 1.minute.from_now.then do |time|
+          delta = time.min % instrument.reserve_interval
+          time - delta.minutes + instrument.reserve_interval.minutes
+        end
```

Comment:
Can you add a comment explaining what's this doing? 

From what I understand, it's trying to get the right minute to start the reservation (ie: 15:05 if current time is 15:02). In that case, I don't think we're supposed to have that restriction for daily booking. Is allowing reservation to start at 15:02 harder?

Reply:
Following the conversation from the other comment, I'm going to leave the reserve_interval to nil and use 1 as default interval for daily reservations if required